code/index.html

```diff
@@ -4,27 +4,95 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <!-- 1. HERO IMAGE -->
+    <header class="hero-img">
+      <picture>
+        <source srcset=".\images\hero_img_mobile3.jpg" media="(max-width: 776px)"/>
+        <source srcset=".\images\hero_img_tablet3.jpg" media="(max-width: 1190px)"/>
+        <img src="./images/hero_img_desktop3.jpg" alt="Hero Image Drums">
```

Comment:
Paths should be written using forward slash and file names + folders should be in kebab-case :)

Reply:
Nice hero!